docs/Features/XMLFeatures.md

```diff
@@ -128,25 +128,7 @@ See [XML Validation](../Validation.md#xml-validation) for more details.
 
 ## Generate Grammar from XML
 
-### Generate XSD from XML
-
-Using an XML file, an XML Schema/XSD file can be generated from the opening tag in the XML file. With the cursor on the first opening tag, use `Ctrl + .` or `Quick Fix...` and select `Generate foo.xsd and bind with *` to create the file in the the same directory.
-
-![GenerateXSDFromXML](../images/Features/GenerateXSDFromXML.gif)
-
-#### Read more
-
-See [Validation with XSD Grammar](../Validation.md#validation-with-xsd-grammar) for more.
-
-### Generate DTD from XML
-
-Using an XML file, a Document Type Definition/DTD file can be generated from the opening tag in the XML file. With the cursor on the first opening tag, use `Ctrl + .` or `Quick Fix...` and select `Generate foo.dtd and bind with *` to create the file in the the same directory.
-
-![GenerateDTDFromXML](../images/Features/GenerateDTDFromXML.gif)
-
-#### Read more
-
-See [Validation with DTD Grammar](../Validation.md#validation-with-dtd-grammar) for more.
+See [Binding with New Grammar](../BindingWithGrammar.md#quick-fix/codeaction) for info on grammar generation.
 
```

Comment:
It would be nice to add another section to this document to list that you can bind to an existing grammar as well. This would serve to make `XMLFeatures.md` into a mostly complete feature list of vscode-xml. For the content, you can just link to the "Binding With Grammar" page like you did with "Generate Grammar".

Reply:
I generalized to a header called "Binding Grammar to an XML Document"